geonode/facets/urls.py

```diff
@@ -0,0 +1,26 @@
+#########################################################################
+#
+# Copyright (C) 2023 OSGeo
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+#########################################################################
+
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path("facets", views.list_facets, name="list_facets"),
```

Comment:
if is used as an api, I suggest keeping it as `/api/2/facets` using the rest framework and using it as an ApiView, the second URL is already handled by DRF itself (you can specify the method to get the detail if needed, or a serializer could be enough)